extensions/wikia/VideoPageTool/VideoPageToolHooks.class.php

```diff
@@ -13,7 +13,7 @@ public static function onArticleFromTitle( &$title, &$article ) {
 
 		if ( $title->isMainPage() && $app->checkSkin( 'oasis' ) ) {
 			$app->wg->SuppressPageHeader = true;
-			$app->wg->SuppressWikiHeader = true;
+			$app->wg->SuppressCommunityHeader = true;
 			$app->wg->SuppressRail = true;
 			$app->wg->SuppressFooter = true;
 			if ( !$app->wg->request->wasPosted() ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)